### ISSUE TITLE
discontinuity in Tleaf with changing windspeed

### ISSUE THREAD

**Issue by CraigBarton9** [cdmuir/tealeaves#12]
I wanted to see how sensitive leaf temperature is to wind speed so kept everything as default and varied windspeed using script below.

require(tealeaves)

leaf_par <- make_leafpar()
enviro_par <- make_enviropar(replace=list(wind=set_units(c(0.1,0.2,0.5,0.6,0.65, 0.7,1,2,3),m/s)))
constants <- make_constants()
pars <- c(leaf_par, enviro_par, constants)

tl<-tleaves(leaf_par, enviro_par, constants, quiet = FALSE, set_units = TRUE)
with(tl,plot(T_leaf-T_air~wind))

there is a discontinuity in the leaf temperature as wind goes from 0.60 to 0.65 m/s. Not sure what is causing it.

**Comment by cdmuir** [comment]
Thanks for your inquiry. I am really busy with teaching right now, but I will try to help soon. My hunch is that it a switch from free to forced convection. The Nusselt number changes abruptly at a cutoff in Reynolds number. Details described in the Appendix of Muir 2019 (https://doi.org/10.1093/aobpla/plz054). I need to double-check this is correct.

**Comment by CraigBarton9** [comment]
Yes I was thinking it would be the change from free to forced convection or from laminar to turbulent flow at the leaf surface. So Reynolds number. Somewhat stochastic and empirical but it does make a big step in estimates of leaf temperature. One almost needs to provide a confidence range for leaf temperature to show that at under certain conditions it is uncertain due to chaos creeping in.
